### PR TITLE
OC-13147: Changes made for ClearFilter to show default view

### PR DESCRIPTION
**Pull request by suthirr-openclinica**

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/viewAllSubjectSDVtmp.jsp [hunk 1]
```diff
@@ -554,6 +554,15 @@
             getItems();
         }).change();
 
+        $('#clear-filter').click(function(){
+                jQuery('#sdv-items').dataTable().fnSortNeutral();
```
Comment (svadla-oc) [inline src/main/webapp/WEB-INF/jsp/viewAllSubjectSDVtmp.jsp:558]: why are you sorting the items here?

Comment (suthirr-openclinica) [reply src/main/webapp/WEB-INF/jsp/viewAllSubjectSDVtmp.jsp:558]: This is what done previously in clearFilter() function (line 421) to remove the filters on each column, So I used the same and added toggle to defaultRadioButton

Comment (svadla-oc) [reply src/main/webapp/WEB-INF/jsp/viewAllSubjectSDVtmp.jsp:558]: why not update the existing clearFilter() function?

Comment (suthirr-openclinica) [reply src/main/webapp/WEB-INF/jsp/viewAllSubjectSDVtmp.jsp:558]: data was not able to accessed from clearFunction(), So I created a new function inside it

Comment (suthirr-openclinica) [reply src/main/webapp/WEB-INF/jsp/viewAllSubjectSDVtmp.jsp]: Altered the code to use clearFilter() method in all places

**Comment by svadla-oc** [comment]
👍

**Pull request merged**